src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -1293,6 +1303,22 @@ public void verifyGlobalNavigation() {
 	  PageObjectLogging.log("verifyGlobalNavigation", "Verified global navigation", true);
   }
 
+  public void openFlagModal() {
+    waitForElementClickableByElement(getEditButton);
+    getEditButton.click();
+    waitForElementByElement(flagButton);
+    flagButton.click();
+    PageObjectLogging.log("openFlagModal", "Verified flag modal", true);
```

Comment:
don't you need to check flag modal was opened? You log "Verified flag modal", but actually it's not verified